unittests.py

```diff
@@ -34,7 +34,11 @@ def tealiumCallback(self, info, success, error=None):
     def test_trackEvent(self):
         t = Tealium('tealiummobile', 'demo', 'dev')
         d = {"foo": "bar"}
-        t.trackEvent('title', t.EVENT_TYPE_VIEW, d, self.tealiumCallback)
+        t.trackEvent(title="test",
+                     data=d,
+                     eventType=Tealium.EVENT_TYPE_VIEW,
+                     tealiumCallback=self.tealiumCallback)
+
         t = None
```

Comment:
Presuming this is needed to clear out the variable?  If so, should be added to the other tests that assign a value to 't'